README.md

```diff
@@ -6,9 +6,23 @@
 [![Nordic ClangFormat](https://img.shields.io/static/v1?label=Nordic&message=ClangFormat&labelColor=00A9CE&color=337ab7)](https://github.com/nrfconnect/sdk-nrf/blob/master/.clang-format)
 [![Zephyr compliance](https://img.shields.io/static/v1?label=Zephry&message=compliance&labelColor=4e109e&color=337ab7)](https://docs.zephyrproject.org/latest/contribute/index.html#coding-style)
 
+It determines NAT timeouts in a cellular networks by sending messages to the [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer) and waiting for the reply.
+
 This application is built with [sdk-nrf](https://github.com/nrfconnect/sdk-nrf).
 
-Used to test NAT timeouts in different networks, sends messages to [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer).
+## Getting started
+
+Please help us learn more about NAT timeouts in cellular networks:
+
+1. Find the latest firmware build in the [releases](https://github.com/NordicSemiconductor/NAT-TestFirmware/releases) and flash it onto your nRF9160 Development Kit.
+1. Put in the SIM card of your choice and power on the kit. The test will start automatically. **Do not change the location of the kit during testing, switching mobile cells must be avoided.**
+1. Optionally: you may connect it via USB to observe the test runs in a terminal.
+1. Wait until the test finish (indicated by the 4 LEDs blinking in a rotating pattern). If the TCP test is still running after 24 hours, you can abort it. We generally assume that more than 24 hours means sufficient power savings are provided from the network for the majority of use case scenarios.
+1. Register an account on <https://cellprobe.thingy.rocks/> and log-in to see your test-results (they are updated every hour).
+1. If your SIM does not show up this could mean that the ICCID is unknown. Please open an issue in the [TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).
```

Comment:
```suggestion
1. If your SIM does not show up after login, it could mean that the ICCID is unknown. In this case, open an issue in the [TestServer repository](https://github.com/NordicSemiconductor/NAT-TestServer/issues/new).
```